src/qt/askpassphrasedialog.cpp

```diff
@@ -116,15 +116,15 @@ void AskPassphraseDialog::accept()
             break;
         }
         QMessageBox::StandardButton retval = QMessageBox::question(this, tr("Confirm wallet encryption"),
-                 tr("Warning: If you encrypt your wallet and lose your passphrase, you will <b>LOSE ALL OF YOUR NAMECOINS</b>!") + "<br><br>" + tr("Are you sure you wish to encrypt your wallet?"),
+                 tr("Warning: If you encrypt your wallet and lose your passphrase, you will <b>LOSE ALL OF YOUR NAMECOINS AND NAMES</b>!") + "<br><br>" + tr("Are you sure you wish to encrypt your wallet?"),
```

Comment:
While we are changing this, would you mind changing `NAMECOINS` to just `COINS` (i.e. "LOSE ALL OF YOUR COINS AND NAMES")?

Reply:
I prefer not to, since upstream uses `BITCOINS` rather than `COINS`.  Maybe it would be useful to get upstream to use a variable for this though (i.e. I think strings like `Bitcoin` are usually factored out into a variable upstream; not sure why they didn't do that here).

Reply:
Well, we use "coins" already in lots of other places (like RPC help texts), so I don't see why this would be different here?  I don't have a strong opinion, though, so if you don't want to make that change, it is fine with me as well.

Reply:
I would prefer not to make that change, but I've filed an [upstream issue](https://github.com/bitcoin-core/gui/issues/380).